src/panels/developer-tools/state/developer-tools-state.js

```diff
@@ -98,6 +98,7 @@ class HaPanelDevState extends EventsMixin(LocalizeMixin(PolymerElement)) {
         ha-tip {
           display: flex;
```

Comment:
Can we remove `display: flex`, I feel like `ha-tip` should handle that?

Reply:
W/o flex:
![image](https://github.com/home-assistant/frontend/assets/71872483/e8c3f845-3879-4af9-8121-3b5f064d07fd)

Reply:
Probably the correct style should be defined for the "standard" ha-tip component. Now it is "block".

Reply:
I can propose a PR for ha-tip component if you agree this way.

Reply:
It will need more than just `display: flex` (the icon changes size for example, and the text is not vertically centered) but yeah, that's fine by me.

Reply:
Tested with `display: flex` for the "ha-tip" component - and I do not like results.

Check this for some iPhone with the current default settings:
![image](https://github.com/home-assistant/frontend/assets/71872483/2f1e3467-4aba-4c36-acd8-a3e4fab5f0b7)

And with these:
```
    display: flex;
    justify-content: center;
    align-items: center;
```
![image](https://github.com/home-assistant/frontend/assets/71872483/fd57fbed-1e4a-42b2-9cb5-6e0c5af99b0e)

Or with this (`align-items: start`):
![image](https://github.com/home-assistant/frontend/assets/71872483/5ff2d3be-f35c-4ec1-a6a4-c32ba69e1355)

The default style looks much nicer!
Here the "tip" label is supposed to be center-aligned (horizontally), and in the "Dev tools -> Set state panel" the tip as displayed along with data which are left-aligned.

So I propose to use the `display: flex` style for Dev tools -> Set state only, not globally.

Another unrelated proposal: let's get rid of the "Tip!" label, the "bulb" icon is self-explanatory (btw this label is hidden in "Dev tools -> Set state"):

![image](https://github.com/home-assistant/frontend/assets/71872483/139852f5-8bf1-4285-b04b-b3f2ecdbc497)